jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/UpdateChecker.java

```diff
@@ -139,11 +150,17 @@ void setDisableUpdateCheck(boolean disableUpdateCheck) {
       }
 
       // Check for update
-      HttpURLConnection connection = (HttpURLConnection) new URL(versionUrl).openConnection();
+      FailoverHttpClient httpClient = new FailoverHttpClient(true, false, log);
+      Response response =
+          httpClient.get(
+              new URL(versionUrl),
+              Request.builder()
+                  .setHttpTimeout(3000)
+                  .setUserAgent("jib " + ProjectInfo.VERSION + " " + toolName)
```

Comment:
I actually don't like our current string format (#1067), but I guess we'll just go with this for consistency.